Attribute/AsDocumentListener.php

```diff
@@ -14,10 +15,28 @@ class AsDocumentListener
 {
     public function __construct(
         public ?string $event = null,
+        /** @deprecated the method name is the same as the event name */
         public ?string $method = null,
+        /** @deprecated not supported */
         public ?bool $lazy = null,
         public ?string $connection = null,
         public ?int $priority = null,
     ) {
+        // phpcs:disable SlevomatCodingStandard.ControlStructures.EarlyExit.EarlyExitNotUsed
+        if ($method !== null) {
+            Deprecation::trigger(
```

Comment:
We used to use Symfony's deprecation mechanisms in ODM, is it no longer the case?

Reply:
Great! I'm still learning.

Reply:
Fixed.